src/views/pages/index.ejs

```diff
@@ -8,7 +8,11 @@
   <link rel="stylesheet" href="css/index.css">
   <meta name="robots" content="noindex">
   <meta name="twitter:card" content="summary_large_image" />
-  <meta property="og:image" content="<%= img %>" />
+  <% if (img.endsWith('.mp4')) { %>
+    <meta property="og:video" content="<%= img %>" />
```

Comment:
<meta name="twitter:player" content="video url">

Reply:
<meta name="twitter:player" content="Video url">

Reply:
you try it

Reply:
if it works i will change it